app/src/main/java/org/simple/clinic/patient/RecentPatient.kt

```diff
@@ -68,18 +65,6 @@ data class RecentPatient(
               AND cancelReason IS NULL
```

Comment:
We don't really need to account for the following anymore.
```
AND remindOn IS 
AND agreedToVisit IS 
AND cancelReason IS NULL
```
This can be removed.

Reply:
Good catch. Done.